aptoide-ui/src/main/java/cm/aptoide/pt/aptoide_ui/textformatter/TextFormatter.kt

```diff
@@ -44,6 +45,11 @@ class TextFormatter {
       return decimalFormatter.format(value)
     }
 
+    fun formatDecimalTwoPlaces(value: Double): String {
```

Comment:
Sergei suggested that we avoid using this module and create an extension function on the `extensions` module or encapsulate the formatting into the class itself.